package.json

```diff
@@ -18,35 +18,44 @@
     "url": "https://github.com/cypress-io/eslint-plugin-cypress/issues"
   },
   "homepage": "https://github.com/cypress-io/eslint-plugin-cypress#readme",
+  "engines": {
+    "node": ">=8.10.0"
+  },
   "peerDependencies": {
-    "eslint": ">= 3.2.1"
+    "eslint": ">= 7.0.0"
```

Comment:
This change makes this a breaking change, since users will have to upgrade their eslint to use it. To ensure this triggers a breaking change, it needs have a commit that follows the [semantic release format](https://github.com/semantic-release/semantic-release#commit-message-format), which is a bit different than triggering a patch or minor release.